code_generator/End2End/Tests/BlittableComponentTests.cs

```diff
@@ -0,0 +1,75 @@
+using Generated.Improbable.TestSchema.Blittable;
+using Improbable.Gdk.Core;
+using NUnit.Framework;
+using Unity.Mathematics;
+
+namespace Improbable.Gdk.CodeGenerator.End2EndTests
+{
+    [TestFixture]
+    public class BlittableComponentTests
+    {
+        private static readonly bool1 Bool1False = new bool1(false);
+        private static readonly bool1 Bool1True = new bool1(true);
+
+        private const int IntValue = 123;
+        private const long LongValue = 5678L;
+        private const float FloatValue = 1.2345f;
+        private const double DoubleValue = 3.14159;
+        private static readonly bool1 BoolValue = Bool1True;
```

Comment:
do we need `BoolValue`? it looks like we can just use `Bool1True`

Reply:
I think for consistency it's nice to just have *Value properties that can be used, if / when these tests get bigger.